ui/raidboss/data/06-ew/trial/endsinger.ts

```diff
@@ -0,0 +1,251 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// @TODO:
+// Interstellar - Position here can be gotten quicker via getCombatants on the chat line 'Tis so lonely between the stars'
+// current trigger doesn't give much time at all to dodge
+// Add phase triggers
+// Final phase triggers
+
+export interface Data extends RaidbossData {
+  storedStar?: { x: number; y: number };
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheFinalDay,
+  timelineFile: 'endsinger.txt',
+  triggers: [
+    {
+      id: 'Endsinger Doomed Stars AoE',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['662E', '6634'], source: 'Doomed Stars', capture: true }),
+      alertText: (data, matches, output) => {
+        const x = parseFloat(matches.x);
+        const y = parseFloat(matches.y);
+
+        data.storedStar = { x, y };
+
+        if (x < 100) {
+          if (y < 100)
+            return output.se?.();
```

Comment:
Can you use `output.se!()` here? When I converted all the triggers to TypeScript, there was some discussion and we decided that output functions were the one place to use non-null assertions just to make them slightly less obnoxious to read.  If in the future TypeScript can understand more about proxies then we won't need it.